torch/csrc/jit/passes/onnx/constant_map.cpp

```diff
@@ -272,6 +294,16 @@ void ConstantValueMap::PrintMaps() {
       std::cout << std::endl;
     }
   }
+  std::cout << std::endl;
+  std::cout << "Print ShapeValue Maps:" << std::endl;
```

Comment:
```suggestion
  std::cout << "shapeValueMap:" << std::endl;
```

Similar change for the other lines above. We don't need to say "Print foo", just "foo" :-)